src/app/models/login/user.model.ts

```diff
@@ -1,13 +1,24 @@
 /**
  * @author Hadi Horani, s144885
+ * @author Anders Wiberg Olsen, s165241
  */
 
+export class LoginModel {
+    username: string;
+    password: string;
```

Comment:
Problemet ved at sætte koden som blot en string er, at selve koden vil blive fremvist i konsollen for den pågældende payload. Der vil komme til at stå: {username: "sXXXXXX", password: "MINKODE"}